js/client-ladder.js

```diff
@@ -52,6 +52,17 @@
 				this.subtleNotifyOnce();
 				break;
 
+			case 'pagescroll':
+				var page = this.$el.get()[0];
+				if (!page) break;
+
+				if (row[1] === 'top') {
+					page.scrollBy(0, -1 * page.scrollHeight);
```

Comment:
This looks like a bleeding-edge API. I would recommend using `scrollTop` instead, which should have better compatibility.

Reply:
```suggestion
					page.scrollTop = 0;
```

Reply:
https://github.com/smogon/pokemon-showdown-client/blob/9bf683df7be1ffb92a242410052256d49f353049/src/battle-log.ts#L328-L332

Yeah, this is how we do it in `battle-log`. It's best to be consistent.